config.md

```diff
@@ -49,6 +49,14 @@ For example, if a configuration is compliant with version 1.1 of this specificat
 }
 ```
 
+### Example (Windows)
+
+```json
+"root": {
+    "path": "\\\\?\\Volume{ec84d99e-3f02-11e7-ac6c-00155d7682cf}"
```

Comment:
The trailing backslash doesn't actually matter. The example I gave was pasted from a breakpoint on a live running container.